### PR TITLE
fix: add missing openapi response schema to /status

### PR DESCRIPTION
**Pull request by Alex-Izquierdo**

### CODE REVIEW

File: src/aap_eda/core/views.py
Comment (jshimkus-rh) [inline src/aap_eda/core/views.py:1]: Might as well use the imported status for the values in HealthView.

Comment (Alex-Izquierdo) [reply src/aap_eda/core/views.py:1]: No follow, sorry.

Comment (jshimkus-rh) [reply src/aap_eda/core/views.py:1]: The HealthView class, in the same file, is using the literals `200` and `500`.  Let's change that to use the same `status` attributes.

Comment (Alex-Izquierdo) [reply src/aap_eda/core/views.py:1]: I think that's out of the scope of this PR.

Comment (jshimkus-rh) [reply src/aap_eda/core/views.py:1]: That then makes the critique the new code is inconsistent with the existing code.

Comment (Alex-Izquierdo) [reply src/aap_eda/core/views.py:1]: There are more inconsistencies like this in the whole app. Do you want to address all them here? Or it is just that file? Do you want to keep the old literal?

Comment (Alex-Izquierdo) [reply src/aap_eda/core/views.py:1]: HealthView changed.

**Review by jshimkus-rh — APPROVED** [review]

**Review by Dostonbek1 — APPROVED** [review]

**Pull request merged**